ThunderRequest/TSCRequestController.m

```diff
@@ -608,7 +608,7 @@ - (void)scheduleDownloadRequest:(TSCRequest *)request on:(NSDate *)beginDate pro
 			NSURLRequest *normalisedRequest = [self backgroundableRequestObjectFromTSCRequest:request];
 			NSURLSessionDownloadTask *task = [welf.backgroundSession downloadTaskWithRequest:normalisedRequest];
             
-            if (@available(iOS 11.0, *)) {
+            if (@available(iOS 11.0, *) && @available(watchOS 4.0, *)) {
```

Comment:
Should this be `and`, or, `or`?

Reply:
Neither of us were right 😆 turns out you can put multiple os' in the @available parameters